vector/src/main/res/values/donottranslate.xml

```diff
@@ -17,7 +17,16 @@
     <string name="ftue_auth_create_account_choose_server_header">Choose your server to store your data</string>
     <string name="ftue_auth_create_account_sso_section_header">Or</string>
     <string name="ftue_auth_create_account_matrix_dot_org_server_description">Join millions for free on the largest public server</string>
-    <string name="ftue_auth_create_account_matrix_dot_org_server_name">matrix.org</string>
     <string name="ftue_auth_create_account_edit_server_selection">Edit</string>
 
+    <string name="ftue_auth_choose_server_title">Choose your server</string>
+    <string name="ftue_auth_choose_server_subtitle">What is the address of your server? Server is like a home for all your data.</string>
+    <string name="ftue_auth_choose_server_entry_hint">Server URL</string>
+    <string name="ftue_auth_choose_server_entry_footer">You can only connect to a server that has already been set up</string>
+    <string name="ftue_auth_choose_server_ems_title">Want to host your own server?</string>
+
+    <string name="ftue_ems_url">https://element.io/ems</string>
```

Comment:
Maybe we should add this to vector-config urls.xml ?

Reply:
https://github.com/vector-im/element-android/pull/5691/commits/5f9d3e107deef76692aaaa73b5daf01ff453ab5b

Reply:
Thanks